### DISCUSSION TITLE
Some Students Unable to Accept Assignments

### DISCUSSION THREAD

**Discussion by JamesonRGrieve-Instruction** [community/community#44415] [GitHub Education] [answered]
I have 2/10 students receiving this error on all assignments in the course. 8/10 can accept normally. Am I missing something obvious here? I've tried creating other assignments with different due dates and I can't get it to work for them.

![0](https://user-images.githubusercontent.com/66742895/212566797-9411c313-7338-4a92-8cba-677e46af5591.png)

**Comment by jd12** [comment]
I've had the same problem for the past few days. it's also happening intermittently for me. Not for all students. I tried accepting the assignments myself and got the same error.

**Comment by jillhubbard** [comment]
I have the same issue. Some students get the message while others do not.
I created a public template and used it when creating my assignment.  Here is a link to my assignment:
https://classroom.github.com/a/j3N-0q-K
2 students received the following message:
"You accepted the assignment, Scrabble but something went wrong. We have not received any updates for this assignment. You may want to reaccept the assignment. "

I can see that the student's github is linked to my classroom but they can not accept the assignment.

**Comment by samus-aran** [comment, marked as answer]
Hi @JamesonG-UAlberta & @jillhubbard 👋 

This seems like a duplicate discussion on a [similar subject matter.](https://github.com/orgs/community/discussions/43783) We will lock this thread and divert the conversation to keep the community discussions boards tidy. Thank you for helping us maintain a productive and tidy community for all our members.
